micro_pumas_utils.F90

```diff
@@ -1341,7 +1342,87 @@ subroutine ice_deposition_sublimation_mg4(t, qv, qi, niic, &
 
   !$acc end data
 end subroutine ice_deposition_sublimation_mg4
-
+            
+subroutine vapor_deposition_onto_snow(t, qv, qs, ns, &
+   precip_frac, rho, dv,qvl, qvi, asn, mu, sc, &
+   vap_deps, vlen)
+
+!INPUT VARS:
+!===============================================
+integer,  intent(in) :: vlen
+real(r8), dimension(vlen), intent(in) :: t
+real(r8), dimension(vlen), intent(in) :: qv
+real(r8), dimension(vlen), intent(in) :: qs
+real(r8), dimension(vlen), intent(in) :: ns
+real(r8), dimension(vlen), intent(in) :: precip_frac
+real(r8), dimension(vlen), intent(in) :: rho
+real(r8), dimension(vlen), intent(in) :: dv
+real(r8), dimension(vlen), intent(in) :: qvl
+real(r8), dimension(vlen), intent(in) :: qvi
+real(r8), dimension(vlen), intent(in) :: asn   ! fall speed parameter for snow
+real(r8), dimension(vlen), intent(in) :: mu
+real(r8), dimension(vlen), intent(in) :: sc
```

Comment:
One quick comment  describing each input parameter will go a long way in the future when using this subroutine.

Reply:
Added